### PR TITLE
PR for #34

### PR DESCRIPTION
**Pull request by idofish**
PR for issue #34

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -44,7 +44,12 @@ async def websocket_endpoint(websocket: WebSocket):
         except WebSocketDisconnect:
             print("Client disconnected.")
             break
-    await websocket.close()
+        # Since we're already handling the specific exception above and logging the exception
+        # we can suppress pylint W0703 - Catching too general exception Exception (broad-except)
+        except Exception as error: # pylint: disable=W0703
+            print(f"ERROR: {error}")
```
Comment (shubham1172) [inline main.py:50]:
```suggestion
            print(f"Error: {error}")
```
We can also make it red-colored using `console.py`.

**Review by shubham1172 — CHANGES_REQUESTED** [review]
Thanks for your PR! I just have a nit.

File: main.py (hunk 1 above)
Comment (idofish) [reply main.py:50]: Should I change the color of the other print? (the one for the client disconnect)

Comment (shubham1172) [reply main.py]: Since it's expected, I'd think of it as an info message instead of an error.

Comment (idofish) [reply main.py]: Added. Had to change the printing in console.py to use f-strings for pylint

**Review by shubham1172 — APPROVED** [review]
LGTM, thank you for the PR!

**Comment by idofish** [comment]
Happy to help :)

**Comment by shubham1172** [comment]
Since it's expected, I'd think of it as an info message instead of an error.

**Pull request merged**